reference/core/ingress-controller.md

```diff
@@ -58,7 +58,7 @@ If you're new to the Ambassador Edge Stack and to Kubernetes, we'd recommend you
             targetPort: https
             ```
 
-### When should I use an `Ingress` instead of annotations or CRDs?
+### When Should I Use `Ingress` Instead of Annotations or CRDs?
```

Comment:
I think "an" should still be there? "Ingress" is a countable noun--either "an `Ingress`" or multiple "Ingresses".

Reply:
addressed